src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const dayPrice = 40;
+  const discount1 = 20;
+  const discount2 = 50;
```

Comment:
Consider using more descriptive names for the constants to indicate their purpose. For instance, `DAILY_RENTAL_RATE`, `SHORT_TERM_DISCOUNT`, and `LONG_TERM_DISCOUNT` would more clearly convey the meaning of these values and their usage in the context of the rental cost calculation.